packages/template-retail-react-app/app/pages/page-not-found/index.jsx

```diff
@@ -80,4 +80,10 @@ const PageNotFound = () => {
     )
 }
 
+PageNotFound.getProps = async ({res}) => {
+    if (res) {
+        res.status(404)
```

Comment:
Should we port this bug fix for V1 too? The issue is also happening there.

Reply:
if you are starting a project, you should be on v2 :)